test/views/payments/LateSubmissionPenaltyViewSpec.scala

```diff
@@ -0,0 +1,200 @@
+      elementText(".govuk-summary-list__row:nth-child(3) > dt") shouldBe "Amount received"
+    }
+
+    "display the amount received" in {
+      elementText(".govuk-summary-list__row:nth-child(3) > dd") shouldBe s"£${lateSubmissionPenaltyModel.amountReceived}"
+    }
+
+    "have the correct heading for the fourth row" in {
+      elementText(".govuk-summary-list__row:nth-child(4) > dt") shouldBe "Left to pay"
+    }
+
+    "display the outstanding amount" in {
+      elementText(".govuk-summary-list__row:nth-child(4) > dd") shouldBe s"£${lateSubmissionPenaltyModel.leftToPay}"
+    }
+
+    "have a pay now button" which {
+
+      "has the correct link text" in {
+        elementText("#content > div > div > a") shouldBe "Pay now"
+      }
+
+
+      "has the correct href" in {
+        element("#content > div > div > a").attr("href") shouldBe
+          "/vat-through-software/make-payment/10055/6/2020/2020-06-06/VAT%20Late%20Submission%20Pen/2020-10-01/CHARGEREF"
+      }
+    }
+
+    "have a link to VAT penalties and appeals" which {
+
+      "has the correct link text" in {
+        elementText("#content > div > div > p:nth-child(4) > a") shouldBe "View your VAT penalties and appeals"
+      }
+
+      "has the correct href" in {
+        element("#content > div > div > p:nth-child(4) > a").attr("href") shouldBe mockConfig.penaltiesFrontendUrl
+      }
+    }
+
+
+    "have a link to the what you owe page" which {
+
+      "has the correct link text" in {
+        elementText("#content > div > div > p:nth-child(5) > a") shouldBe "Return to what you owe"
+      }
+
+      "has the correct href" in {
+        element("#content > div > div > p:nth-child(5) > a").attr("href") shouldBe whatYouOweLink
+      }
+    }
+  }
+
+
+  "Rendering the Late Submission Penalty page for an agent" should {
+
+    lazy val view = injectedView(lateSubmissionPenaltyModel, Html(""))(request, messages, mockConfig, agentUser)
+    lazy implicit val document: Document = Jsoup.parse(view.body)
+
+    "have the correct document title" in {
+      document.title shouldBe "Late submission penalty - Your client’s VAT details - GOV.UK"
+    }
+
+    "have a link to VAT penalties and appeals" which {
+
+      "has the correct link text" in {
+        elementText("#content > div > div > p:nth-child(3) > a") shouldBe "View your client’s VAT penalties and appeals page"
+      }
+
+      "has the correct href" in {
+        element("#content > div > div > p:nth-child(3) > a").attr("href") shouldBe mockConfig.penaltiesFrontendUrl
+      }
+    }
+
+    "have a link to the what you owe page" which {
+
+      "has the correct link text" in {
+        elementText("#content > div > div > p:nth-child(4) > a") shouldBe "Return to what your client owes"
+      }
+
+      "has the correct href" in {
+        element("#content > div > div > p:nth-child(4) > a").attr("href") shouldBe whatYouOweLink
+      }
+    }
+
+    "not render breadcrumbs" in {
```

Comment:
Considering you're testing when breadcrumbs/pay button are not visible for an agent, does it make sense to throw in a test for the principal user to confirm back link is not visible?